src/Logging/RpcLogger.cs

```diff
@@ -55,6 +56,8 @@ public void Log(bool isUserOnlyLog, LogLevel logLevel, string message, Exception
                 };
 
                 _msgStream.Write(logMessage);
+
+                outputLogHandler.LogToHandlers(logLevel.ToString(), message, exception);
```

Comment:
Consider passing the `LogLevel` value directly, and converting it to a string later, when a string is actually needed. Hopefully, this will let you replace `Action<string, string, Exception>` with `Action<LogLevel, string, Exception>`, which is way more understandable.

Reply:
Oh, I see, the handler is actually in the module, I didn't realize that. Ok, thank you.